# SPDX-License-Identifier: GPL-3.0-only
# eurorack f4cfa7697095a28d292bff758beb51deb49b5a86
--- a/plaits/dsp/fm/algorithms.h
+++ b/plaits/dsp/fm/algorithms.h
@@ -195,6 +195,7 @@
   DISALLOW_COPY_AND_ASSIGN(Algorithms);
 };
 
+#ifndef _MSC_VER
 /* static */
 template<> const uint8_t Algorithms<4>::opcodes_[][4];  // From DX100
 
@@ -206,6 +207,7 @@
 
 /* static */
 template<> const Algorithms<6>::RendererSpecs Algorithms<6>::renderers_[];
+#endif
 
 }  // namespace fm
 
